### PR TITLE
User import & Login restyle

### PR DESCRIPTION
**Pull request by RobLoweMOJ**

### CODE REVIEW

File: webpack.mix.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const mix_ = require('laravel-mix');
+
+mix_.setPublicPath('./dist')
+  .sass('./assets/scss/login.scss', 'css/login.css')
+  .options({
+    processCssUrls: false
+  });
+
```
Comment (brown-a2) [inline webpack.mix.js:8]: I know the file was pretty small to start with so maybe not worth it but isn't there a function like .minify('dist/css/login.css') you can use to minify the css?

File: inc/import-users.php [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    // Define variables to store form data and error messages
+    $json_data = '';
+    $errors = [];
+    $added_users = 0;
+    $failed_users = 0;
+
+    // Check if the form is submitted
+    if (isset($_POST['submit'])) {
+        // Retrieve form data
+        $json_data = stripslashes($_POST['json_data']);
```
Comment (brown-a2) [inline inc/import-users.php:34]: I guess it is data we are adding but is it worth adding in sanitize_text_field() so `$json_data = sanitize_text_field(stripslashes($_POST['json_data']));
`?

File: inc/import-users.php [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+        // Validate JSON data
+        if (empty($json_data)) {
+            $errors[] = 'JSON Data is required.';
```
Comment (brown-a2) [inline inc/import-users.php:38]: Optional but doesn't wp like echoing translated strings like  $errors[] = esc_html__('JSON Data is required.', 'hale-components'); ? I guess we're also never doing translations with this so 🤷‍♂️

**Review by brown-a2 — APPROVED** [review]

File: inc/import-users.php (hunk 2 above)
Comment (RobLoweMOJ) [reply inc/import-users.php:34]: orignally had that but it then failed to parse as JSON

**Pull request merged**